### PR TITLE
Use canola oil instead of Vagetable oil

### PR DESCRIPTION
**Pull request by Suparna-here**
pull Request

### CODE REVIEW

File: cake-recipe.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canola oil
```
Comment (ghost) [inline cake-recipe.txt]: increase quantity of oil

Comment (ghost) [reply cake-recipe.txt]: recommended

**Comment by Suparna-here** [comment]
Please have a look

**Comment by ghost** [comment]
deleted different-oil

**Pull request merged**